src/androidTest/java/de/dennisguse/opentracks/content/provider/CustomContentProviderUtilsTest.java

```diff
@@ -766,4 +766,12 @@ private void insertTrackWithLocations(Track track) {
         contentProviderUtils.insertTrack(track);
         contentProviderUtils.bulkInsertTrackPoint(track.getLocations().toArray(new Location[0]), track.getLocations().size(), track.getId());
     }
+
+    @Test
+    public void testFormatIdListForUri() {
+        Assert.assertEquals("", ContentProviderUtils.formatIdListForUri(new long[]{}));
+        Assert.assertEquals("12", ContentProviderUtils.formatIdListForUri(new long[]{12}));
+        Assert.assertEquals("42,43,44", ContentProviderUtils.formatIdListForUri(new long[]{42,43,44}));
```

Comment:
Can  you applying autoformatting?
(I guess, at the moment the defaults of AndroidStudio are used).